models/CreateNewPractitionerRequest.js

```diff
@@ -0,0 +1,41 @@
+/*
+Practitioner {
+  family : string
+  given : string
+  suffix : string
+  phone : string
+  email : string
+  gender : enum (male, female, other, unknown)
+  birthdate : date
+  address : {
+    line: string
+    city: string
+    state: string
+    postalCode: string
+    country: string
+  }
+  qualificationCode : enum (BN, CANP, CER, CMA, CNP, CNS, CRN, EMT, RN, RMA, PN, RPH)
```

Comment:
We need the qualification identifier as well

Reply:
I'll add it in